### PR TITLE
setup/install.asciidoc

### PR DESCRIPTION
**Pull request by daxiangxiaoyi**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md)?
- If submitting code, have you included unit tests that cover the changes?
- If submitting code, have you tested and built your code locally prior to submission with `npm test && npm run build`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.

### CODE REVIEW

**Comment by daxiangxiaoyi** [comment]
@node @wuhaixing @ @cch123

File: docs/setup/install.asciidoc [hunk 1]
```diff
@@ -1,42 +1,35 @@
 [[install]]
-== Installing Kibana
+== 安装 Kibana
 
-NOTE: Starting with version 6.0.0, Kibana only supports 64 bit operating systems.
+注意：从 6.0.0 版本开始，Kibana 只支持 64 位的系统。
```
Comment (howardhuanghua) [inline docs/setup/install.asciidoc:4]: NOTE部分不要翻译，翻译之后编译的时候会报错。

File: docs/setup/install.asciidoc [hunk 2]
```diff
@@ -1,42 +1,35 @@
...
 
-Kibana is provided in the following package formats:
+Kibana 提供以下格式的安装包：
 
 [horizontal]
 `tar.gz`/`zip`::
 
-The `tar.gz` packages are provided for installation on Linux and Darwin and are
-the easiest choice for getting started with Kibana.
+'tar.gz'包用来在 Linux 和 Darwin 系统下安装，而且也是开始 Kibana 最简单的一种选择。
```
Comment (howardhuanghua) [inline docs/setup/install.asciidoc:11]: tar.gz前后的顿号不能改成单引号，不然格式会丢失，且前后需要空格

File: docs/setup/install.asciidoc [hunk 3]
```diff
@@ -1,42 +1,35 @@
...
 +
-The `zip` package is the only supported package for Windows.
+‘zip’包是唯一支持 Windows 系统的安装包。
```
Comment (howardhuanghua) [inline docs/setup/install.asciidoc:13]: zip前后需要保留顿号及空格

File: docs/setup/install.asciidoc [hunk 4]
```diff
@@ -1,42 +1,35 @@
...
 +
-<<targz>> or <<windows>>
+<<targz>> 或者 <<windows>>
 
 `deb`::
 
-The `deb` package is suitable for Debian, Ubuntu, and other Debian-based
-systems.  Debian packages may be downloaded from the Elastic website or from
-our Debian repository.
+Debian包用来在 Debian，Ubuntu，和其他基于 Debian 的系统下安装，Debian 包可以从 Elastic 官网或者我们的 Debian 仓库中下载。
```
Comment (howardhuanghua) [inline docs/setup/install.asciidoc:19]: `deb`  需要保留格式

File: docs/setup/install.asciidoc [hunk 5]
```diff
@@ -1,42 +1,35 @@
...
 +
 <<deb>>
 
 `rpm`::
 
-The `rpm` package is suitable for installation on Red Hat, Centos, SLES,
-OpenSuSE and other RPM-based systems.  RPMs may be downloaded from the
-Elastic website or from our RPM repository.
+'rpm'包用来在 Red Hat，Centos，SLES，OpenSuSe 以及其他基于 RPM 的系统下安装。RPM 包可以从 Elastic 官网或者我们的RPM仓库下载。
```
Comment (howardhuanghua) [inline docs/setup/install.asciidoc:25]: rpm 同上需要保留格式，后面的 RPM 前后需要空格

File: docs/setup/install.asciidoc [hunk 6]
```diff
@@ -1,42 +1,35 @@
...
 <<rpm>>
 
 `docker`::
 
-An image is available for running Kibana as a Docker container. It ships
-with https://www.elastic.co/products/x-pack[X-Pack] pre-installed and is
-available from the Elastic
-Docker registry.
+一个镜像用于在 Docker 容器中运行 Kibana。
+https://www.elastic.co/products/x-pack[X-Pack] 预安装包附带 Kibana Docker 镜像，并且可以从 Elastic Dockeer 仓库中获得。
```
Comment (howardhuanghua) [inline docs/setup/install.asciidoc:32]: 后面的Dockeer有错别字，应该是Docker.

File: docs/setup/install.asciidoc [hunk 7]
```diff
@@ -45,10 +38,10 @@ see {xpack-ref}kibana.html[Using Kibana with X-Pack Security] for additional set
 instructions.
 
 
-include::install/targz.asciidoc[]
+include::targz包安装[]
```
Comment (howardhuanghua) [inline docs/setup/install.asciidoc:41]: 引用部分不需要翻译，下同

**Review by howardhuanghua — COMMENTED** [review]
Setup第一次review

**Comment by howardhuanghua** [comment]
分支和review的标题后续请按照文件路径来统一命名。例如该章节应该命名为：setup/install.asciidoc

**Pull request closed without merging**